.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,3 +14,9 @@
 ## Test Plan
 
 <!-- What demonstrates that your implementation is correct? -->
+
+## Docs
+
+<!-- If you're submitting a nursery rule/action, the documentation is part of the code -->
```

Comment:
Should add something about how the website will automatically be updated, so no docs PR is necessary.